ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2

```diff
@@ -7,11 +7,13 @@ After=database.service
 User=root
 {% if sonic_hwsku == 'ACS-MSN2700' %}
 ExecStartPre=/etc/init.d/sxdkernel start
+ExecStartPre=/usr/bin/mst start
```

Comment:
can you check the /etc/init.d/mst in the mft package? I think it automatically start mst